todo_app/data/trello_items.py

```diff
@@ -0,0 +1,101 @@
+from flask import session
+import os
+import requests
+
+def get_lists():
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+    board_id = os.getenv("TRELLO_BOARD_ID")
+    api_key = os.getenv("TRELLO_API_KEY")
+    api_token = os.getenv("TRELLO_API_TOKEN")
+
+    url = f"https://api.trello.com/1/boards/{board_id}/lists"
+
+    query = {"key": api_key, "token": api_token, "cards": "open"}
+
+    response = requests.request("GET", url, params=query)
+
+    return response.json()
+
+
+def get_item(id):
+    """
+    Fetches the saved item with the specified ID.
+
+    Args:
+        id: The ID of the item.
+
+    Returns:
+        item: The saved item, or None if no items match the specified ID.
+    """
+    items = get_items()
```

Comment:
`get_items()` is not defined, nor is the `items` variable used - this whole line can be deleted.